clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/chunking/ChunkingUtils.java

```diff
@@ -320,8 +384,9 @@ private static <VALUE, CHUNKS_CONTAINER> VALUE getFromStorage(
       // optimize large value retrieval in the future, it's unclear whether the concurrent retrieval approach
       // is optimal (as opposed to streaming the response out incrementally, for example). Since this is a
       // premature optimization, we are not addressing it right now.
+      byte[] chunkKey = chunkedValueManifest.keysWithChunkIdSuffix.get(chunkIndex).array();
       byte[] valueChunk =
-          store.get(partition, chunkedValueManifest.keysWithChunkIdSuffix.get(chunkIndex).array(), skipCache);
+          isRmdValue ? store.getReplicationMetadata(partition, chunkKey) : store.get(partition, chunkKey, skipCache);
```

Comment:
The `skipCache` option can be removed since it has caused an incident in prod.
Can you double confirm it? @majisourav99

Reply:
Yes, its not used anymore in prod.

Reply:
Are you suggesting we remove this feature completely here?

Reply:
Yeah, I think we can.